auth/policies.go

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package auth
+
+import (
+	"context"
+
+	acl "github.com/ory/keto/proto/ory/keto/acl/v1alpha1"
+)
+
+// PolicyService represents a authorization service. It exposes
+// functionalities through `auth` to perform authorization.
+type PolicyService interface {
```

Comment:
For `PolicyCommunicator`, `Policy` prefix is not necessary. It can be omitted.

For `PolicyService`, if I omit the Policy prefix, it would conflict with the [auth package's Service ](https://github.com/mainflux/mainflux/blob/f212b064c5de6438746cf180589a6afdf193ced0/auth/service.go#L95). Maybe it'd be better if I rename to `Authz` instead of `PolicyService`.

Reply:
Avoid `PolicyService` and find a shorter name for that inteface. Also, Interface names should be something like `Authorizer` or something...

Reply:
@dusanb94 what do you think about this?

Reply:
Since there is more than one method (and will probably be even more), `Authorizer` is not necessarily the best name. I think that the old name `Authz` (since this interface is really all about authorization) fits the best.

Reply:
OK, good. `Authz` it is.